lib/ansible/modules/files/archive.py

```diff
@@ -49,8 +50,8 @@
 author:
 - Ben Doherty (@bendoh)
 notes:
-    - requires tarfile, zipfile, gzip, and bzip2 packages on target host
-    - can produce I(gzip), I(bzip2) and I(zip) compressed files or archives
+    - requires tarfile, zipfile, gzip, lzma and bzip2 packages on target host
```

Comment:
Since the import of `lzma` is conditional and doesn't result in a failure if it's not found, this piece of documentation should reflect that the `lzma` module is optional and only required for `xz` compression.

Reply:
done